sessionctx/variable/sysvar.go

```diff
@@ -565,11 +566,7 @@ var defaultSysVars = []*SysVar{
 		return nil
 	}},
 	{Scope: ScopeGlobal, Name: TiDBEnableTelemetry, Value: BoolToOnOff(DefTiDBEnableTelemetry), Type: TypeBool},
-	{Scope: ScopeGlobal, Name: TiDBEnableHistoricalStats, Value: Off, Type: TypeBool, GetGlobal: func(s *SessionVars) (string, error) {
-		return getTiDBTableValue(s, "tidb_enable_historical_stats", Off)
-	}, SetGlobal: func(s *SessionVars, val string) error {
-		return setTiDBTableValue(s, "tidb_enable_historical_stats", val, "Current historical statistics enable status")
-	}},
```

Comment:
This is a bug. See: https://github.com/pingcap/tidb/issues/30821

Unfortunately it needs to be fixed for this refactor to work, since otherwise it causes contention from each tidb-peer trying to read/write to the mysql.tidb table.